mediapipe/examples/desktop/face_mesh_dll/face_mesh_lib.h

```diff
@@ -79,8 +86,12 @@ extern "C" {
 #endif
 
 DLLEXPORT MPFaceMeshDetector *
-MPFaceMeshDetectorConstruct(int numFaces, const char *face_detection_model_path,
-                            const char *face_landmark_model_path);
+MPFaceMeshDetectorConstruct(int numFaces,
+    const char *face_detection_model_path,
+    const char *face_landmark_model_path,
+    bool with_attention = true,
+    const char* face_landmark_model_with_attention_path = "mediapipe/modules/face_landmark/face_landmark_with_attention.tflite"
+    );
```

Comment:
Please, add default values for `face_detection_model_path` and `face_landmark_model_path` as well.

Also, maybe change the order, variables first, then model paths.

Btw:
```suggestion
MPFaceMeshDetectorConstruct(int numFaces,
    const char *face_detection_model_path,
    const char *face_landmark_model_path,
    bool with_attention = true,
    const char* face_landmark_model_with_attention_path = "mediapipe/modules/face_landmark/face_landmark_with_attention.tflite");
```

Reply:
Done